tianshou/policy/base.py

```diff
@@ -94,8 +94,7 @@ def forward(self, batch: Batch,
             # some code
             return Batch(..., policy=Batch(log_prob=dist.log_prob(act)))
             # and in the sampled data batch, you can directly call
```

Comment:
```suggestion
            # and in the sampled data batch, you can directly use
```

``call`` means to call a function.